src/multiselectMixin.js

```diff
@@ -599,6 +599,8 @@ export default {
       /* istanbul ignore else */
       if (this.disabled) return
       /* istanbul ignore else */
+      if (this.option.$isDisabled) return
```

Comment:
This is causing an error when backspace is pressed when nothing is selected.
Consider adding a condition to check if 'option' is undefined at [this line](https://github.com/tarun1793/vue-multiselect/blob/285dfdc5010a469822fe06f96fa11e19bb79c3e7/src/multiselectMixin.js#L523)

Not a severe issue. But I don't like to see my console red.
![error](https://user-images.githubusercontent.com/26731727/55068212-4b06ad00-50a7-11e9-9de0-7de08d47bb9c.png)